hunter_base/include/hunter_base/hunter_messenger.hpp

```diff
@@ -352,6 +454,30 @@ class HunterMessenger {
       phi_i *= -1.0;
     return k*phi_i;
   }
+
+  double AngelVelocity2Angel(double& linear_vel, double& angular_vel, double &radius)
```

Comment:
type, should be `Angle`

Reply:
Hi @marc-hanheide,  this was already there,